src/Components/FieldDateTime/FieldDateTime.js

```diff
@@ -388,7 +388,7 @@ class FieldDateTime extends React.PureComponent {
             .minutes(59)
         : null);
 
-    const classes = classNames('field-text', className);
+    const classes = classNames('field-text', 'relative', className);
```

Comment:
Can you explain how this fixes the issue? this seems like a very broad fix that could have bigger impacts

Reply:
@osbornm I tried to contain it to just this component, since Visually hidden is using absolute positioning and there's no positioning between Label and the AppContentPane in Dashboard, it's position is being set to the parent component and pushed off the edge of the page.

Reply:
@osbornm I could just add the style instead of using the VisuallyHidden component and change the position?

Reply:
The example you gave is specific to the customer portal. How does this change effect using this component not in that specific case?

Reply:
I added Pebble stand alone

Reply:
Moved the positioning down to the Label level and set the content of label to visually hidden